tavery-ManagementConsole:RN Bug Batch (#94715)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

[id="ocp-release-note-management-console-bug-fixes_{context}"]
==== Management Console

> * Previously, the Cluster Settings page would not properly render during a cluster update if the cluster's ClusterVersion lacked a `Completed` update. With this update, the Cluster Setting page properly renders even if the ClusterVersion lacks a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-49839[OCPBUGS-49839])

Reviewer: Hi @tedaveryredhat . Thanks for raising this PR. Are you still adding bugs for management console to this PR? I see more bugs that need release notes, but maybe they were only wordsmithed today. Would you be OK to add them. See the [filter](https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12347491#SIGwKWmOqDAaIiGh1DSCAiYg6oaN4RY8HFpgaAw2CBElyBoBgDBga6GAtG0HT+ogCErGsyGEN+BWVIilR7PMcS+uOQx2tg2Aop6VhAA) here.

* Previously, the link to OperatorHub in the *Installed Operators* page would trigger a hard reload. With this release, this link no longer triggers a hard reload. (link:https://issues.redhat.com/browse/OCPBUGS-54536[OCPBUGS-54536])

* Previously, selecting *All Projects* from the project picker while on the *Create VolumeSnapshot* page resulted in a page not found error. With this release, the VolumeSnapshot list page is correctly displayed. (link:https://issues.redhat.com/browse/OCPBUGS-53227[OCPBUGS-53227])

* Previously, there was incorrect logic for calculating the pod container count causing it to be inaccurate. With this release, the `Ready` and `Started` status in count logic was added so the correct pod container count is displayed, which is consist with `oc` CLI. (link:https://issues.redhat.com/browse/OCPBUGS-53118[OCPBUGS-53118])

> * Previously, the Select above the Node Logs did not close when opened, unless the Select's toggle was clicked again or one of the Select's menu items was clicked. With this release, the Select closes after clicking outside of the menu or by pressing the appropriate key on the keyboard. (link:https://issues.redhat.com/browse/OCPBUGS-52316[OCPBUGS-52316])

Reviewer 2: ```suggestion
* Previously, the *Select* menu above the *Node Logs* section did not close when opened, unless the *Select* menu's toggle was clicked again or one of the menu items was clicked. With this release, the *Select * menu closes after clicking outside of the menu or by pressing the appropriate key on the keyboard. (link:https://issues.redhat.com/browse/OCPBUGS-52316[OCPBUGS-52316])
```

> * Previously, the shared Timestamp component referenced an undefined property when calculating relative times. As a result, most times displayed in the console were not correctly displaying relative strings such as "Just now" or "Less than a minute ago". With this release, the issue is fixed and relative time strings are correctly rendered in the console. (link:https://issues.redhat.com/browse/OCPBUGS-51202[OCPBUGS-51202])

Reviewer 2: ```suggestion
* Previously, the shared timestamp component referenced a undefined property when calculating relative times. As a result, most times displayed in the console were not correctly displaying relative strings such as `Just now` or `Less than a minute ago`. With this release, the issue is fixed and relative time strings are correctly rendered in the console. (link:https://issues.redhat.com/browse/OCPBUGS-51202[OCPBUGS-51202])
```

> * Previously, the observe menu only displayed based on the current user and console configuration for monitoring. This caused other items added by observability plugins to be hidden. With this release, the observe menu also displays items from different observability plugins. (link:https://issues.redhat.com/browse/OCPBUGS-50693[OCPBUGS-50693])

Reviewer 2: ```suggestion
* Previously, the *Observe* menu only displayed based on the current user and console configuration for monitoring. This caused other items added by observability plugins to be hidden. With this release, the *Observe* menu also displays items from different observability plugins. (link:https://issues.redhat.com/browse/OCPBUGS-50693[OCPBUGS-50693])
```
I also think Observability plugins might need to be capitalized but not 100%.

> * Previously, when logging in to the console for the first time, automatic perspective detection caused the console to ignore the specific URL path that the user clicked to get onto the console, and instead load a different page. With this release, the current path is followed even when automatic perspective switching occurs. (link:https://issues.redhat.com/browse/OCPBUGS-50650[OCPBUGS-50650])

Reviewer 2: I wonder if we should update this to just say "with this release the correct path is followed? Since perspectives unified.

* Previously, there was an issue when creating new tabs in the horizontal navigation present in the web console from plugins. With this release, you can use plugins to create tabs on the web console horizontal navigation. (link:https://issues.redhat.com/browse/OCPBUGS-49996[OCPBUGS-49996])

> * Previously, the Cluster Settings page would not properly render during a cluster update if the cluster's ClusterVersion lacked a `Completed` update. With this release, the Cluster Setting page properly renders even if the ClusterVersion lacks a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-49839[OCPBUGS-49839])

Reviewer: Note should match the 4.18 version on https://issues.redhat.com/browse/OCPBUGS-49921

* Previously, the links on the CLI downloads page were not sorted by operating system. With this release, the links are sorted by their operating system in alphabetical order. (link:https://issues.redhat.com/browse/OCPBUGS-48413[OCPBUGS-48413])

* Previously, multiple external link icons could appear in the primary *Action* button of the *OperatorHub* modal. With this release, only a single external link icon appears. (link:https://issues.redhat.com/browse/OCPBUGS-46555[OCPBUGS-46555]) 

> * Previously, clicking the `Don't show again` link in the Lightspeed modal did not correctly navigate to the general *User Preference* tab when one of the other *User Preference* tabs was displayed. After this update, clicking the `Don't show again` link correctly navigates to the general *User Preference* tab. (link:https://issues.redhat.com/browse/OCPBUGS-46511[(*OCPBUGS-46511)] 

Reviewer 2: oh and there is an attribute for lightspeed too.
ols: OpenShift Lightspeed
ols-official: Red{nbsp}Hat OpenShift Lightspeed
